openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingUpdate.tsx

```diff
@@ -63,7 +46,6 @@ const AtomicTestingUpdate: FunctionComponent<Props> = ({
       onUpdateInject={onUpdateAtomicTesting}
       injectId={atomic.inject_id}
       isAtomic
-      teamsFromExerciseOrScenario={teams?.filter((team: TeamStore) => !team.team_contextual) ?? []}
     />
```

Comment:
I think here we lost the teams of atomic testing

Reply:
Fixed